### ISSUE TITLE
Add credit for Social Safety Net image for redesigned homepage

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#3416]
### Overview
We need to add a credit for a new image used in the redesigned version of the homepage so that the Credits webpage is up to date.

### Details
From the Homepage Redesign Credits in Figma, here is a screenshot of the credit's information for reference:
  <details>
    <summary>Click here to see the screenshot of the Social Safety Net credit in Figma</summary>
    <img src="https://user-images.githubusercontent.com/31293603/180343509-2ff6f6e8-6f5b-4e9f-8f8d-bc542cf61e29.png" alt="">
  </details>

### Action Items
- [x] **Feature Branch:** You will be working off a feature branch called `feature-homepage-launch`. Please follow ["Step 1: Create your branch" in How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch#step-1-create-your-branch) wiki article.
- [x] In `_data/internal/credits` directory, create a file called `social-safety-net.yml`
- [x] In the file you just created, copy and paste the following:
```
---
# The title from the source provider
title: Social Safety Net
# The url where you can get direct access to the image on the provider site
title-link: https://unsplash.com/photos/lyiKExA4zQA
# The type of content - image, audio, or video 
content-type: image
# The page it's used on
used-in: Homepage
# Name of the artist
artist: Austin Kehmeier
# Name of the image database you found it on
provider: Unsplash
# Please give the general site URL, not the link to the image
provider-link: https://unsplash.com/
# Location in our GitHub repo
image-url: /assets/images/program-areas/social-safety-net.png
# Alt text for the image on the Credits page
alt: ''
# If an image was altered, please give the name of the HfLA designer that altered the image
altered-by: Kristine Eudey
---
```
- [x] To update your topic branch, please follow ["Step 2: Update your branch with upstream changes" in How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch#step-2-update-your-branch-with-upstream-changes) wiki article.
- [x] The pull request created from this issue should be merged into `feature-homepage-launch`. Please follow ["Step 3: Create a pull request" in How to work off of a feature branch](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch#step-3-create-a-pull-request) wiki article
- [x] Once the pull request associated with this issue is approved and merged, please update and edit epic #3130 by
  - [x] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please do the following in issue #3130:
    - [ ] Move it to the New Issue Approval column
    - [ ] Remove the "Dependency" label from it
    - [ ] Add a "ready for dev lead" label to it    

### Resources/Instructions
- [Credits webpage](https://www.hackforla.org/credits/)
- [Homepage Redesign Credits in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=17951%3A122724)
- [How to work off of a feature branch wiki page](https://github.com/hackforla/website/wiki/How-to-work-off-of-a-feature-branch)
- [Credits yml file template for Iconography and Imagery](https://github.com/hackforla/website/wiki/Credits-yml-file-template-for-Iconography-and-Imagery)
- This issue is tracked in the epic #3130.

**Comment by JessicaLucindaCheng** [comment]
This issue would usually have the milestone 01. Compliance. However, it's specifically for the homepage launch, which has not happened yet.

**Comment by JessicaLucindaCheng** [comment]
For this credit, the link for the image is: https://unsplash.com/photos/lyiKExA4zQA
  <details>
    <summary>Click here to see the image from Unsplash</summary>
    <img src="https://user-images.githubusercontent.com/31293603/180583999-965a026a-2fdc-461d-8355-a553334b31cf.jpg" alt="">
  </details>

It doesn't look exactly like the contents of this image we are using: 
  <details>
    <summary>Click here to see the screenshot of the marked up Social Safety Net credit in Figma</summary>
    <img src="https://user-images.githubusercontent.com/31293603/180583969-12a0d498-a800-495b-b70d-8ce0e8268f8d.png" alt="">
  </details>

Ans from Bonnie: Add an altered-by field and give credit to the HfLA designer that altered the image.

A separate issue will handle displaying the logic for this to show up on the Credits webpage.

**Comment by JessicaLucindaCheng** [comment]
Note: The "Altered by" field will not show up for the credit when you view the Credits page using Docker. The code to have the "Altered by" field viewable will be implemented in a separate issue.

**Comment by SZwerling** [comment]
Progress: I self-assigned this issue yesterday. I will start today.
Blockers:
Availability:  I will have a few hours including today and this weekend.
ETA: Today or tomorrow. (12/2/22 or 12/3/22)

**Comment by ExperimentsInHonesty** [comment]
@JessicaLucindaCheng is the other issue you mentioned in a [comment on 2022-12-01](https://github.com/hackforla/website/issues/3416#issuecomment-1333602077) have an issue number yet?

> Note: The "Altered by" field will not show up for the credit when you view the Credits page using Docker. The code to have the "Altered by" field viewable will be implemented in a separate issue.

**Comment by JessicaLucindaCheng** [comment]
@ExperimentsInHonesty 
- Yes, #3775
